<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classlist_6</class>
 <widget class="QMainWindow" name="classlist_6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>classlist_6</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>791</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>#</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Class ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Number of Students</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="view_class">
       <property name="text">
        <string>VIEW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_class">
       <property name="text">
        <string>EDIT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_class">
       <property name="text">
        <string>REMOVE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addNewClass">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>470</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD NEW CLASS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
